.github/workflows/ci.yml

```
@@ -41,6 +41,9 @@ jobs:
       - name: "Validate Composer dependencies"
         run: "composer validate"
 
+      - name: "Check composer.json normalisation"
```

```suggestion
      - name: "Check composer.json normalization"
```

we speak american here